docs/plugins/redirects.asciidoc

```diff
@@ -145,3 +145,17 @@ include::redirects.asciidoc[tag=gcs-repo-migration]
 === Google Cloud Storage repository plugin
 
 include::redirects.asciidoc[tag=gcs-repo-migration]
+
+[role="exclude",id="ingest-attachment"]
+=== Ingest Attachment plugin
+
+// tag::ingest-attachment-migration[]
+The Ingest Attachment plugin is now included in {es}.
+See the {ref}/ingest-attachment.html[Ingest Attachment] processor.
```

Comment:
> See the {ref}/ingest-attachment.html[Ingest Attachment] processor.

In my local tests, this URL fails. I think it's because it should be {ref}/attachment.html per the changes in this PR.